android/src/main/java/com/smileidentity/react/Mapper.kt

```diff
@@ -153,23 +154,44 @@ fun ReadableMap.toEnhancedKycRequest(): EnhancedKycRequest {
     },
     consentInformation = getMapOrDefault("consentInformation", null)?.toConsentInfo() ?: run {
       ConsentInformation(
-        consentGrantedDate = getCurrentIsoTimestamp(),
-        personalDetailsConsentGranted = false,
-        contactInfoConsentGranted = false,
-        documentInfoConsentGranted = false
+        consented = ConsentedInformation(
+          consentGrantedDate = getCurrentIsoTimestamp(),
+          personalDetails = false,
+          contactInformation = false,
+          documentInformation = false
+        )
       )
     },
   )
 }
 
 fun ReadableMap.toConsentInfo(): ConsentInformation {
+  val consentGrantedDate = getStringOrDefault("consentGrantedDate", null) ?: getCurrentIsoTimestamp()
```

Comment:
```suggestion
  val consentGrantedDate = getStringOrDefault("consentGrantedDate", getCurrentIsoTimestamp())
```
Is that what we want to achieve here?